Doc/library/sqlite3.rst

```diff
@@ -937,13 +914,22 @@ Connection objects
 
    .. method:: setlimit(category, limit, /)
 
-      Set a connection runtime limit. *category* is the limit category to be
-      set. *limit* is the new limit. If the new limit is a negative number, the
-      limit is unchanged.
-
+      Set a connection runtime limit.
       Attempts to increase a limit above its hard upper bound are silently
-      truncated to the hard upper bound. Regardless of whether or not the limit
-      was changed, the prior value of the limit is returned.
+      truncated to the hard upper bound.
+      Regardless of whether or not the limit was changed,
+      the prior value of the limit is returned.
```

Comment:
Reduces the diff slightly

```suggestion
      truncated to the hard upper bound. Regardless of whether or not the limit
      was changed, the prior value of the limit is returned.
```

Reply:
Fair enought; I'll save the reflow to when I'm relocating these ;)

Reply:
I guess I agree with the school of thought that says it's useful to preserve git history in cases like this, where the effect of the change isn't user-visible. But, no strong feelings either way :)

Reply:
I do agree for PRs like this :) But for relocations, it is IMO less important; the git history and the diff will be "ruined" anyway.

Reply:
> But for relocations, it is IMO less important; the git history and the diff will be "ruined" anyway.

Oh, we agree on that!